client/state/login/selectors.js

```diff
@@ -214,7 +207,7 @@ export const isSocialAccountCreating = ( state ) =>
  * @returns {?string}         Username of the created social account
  */
 export const getCreatedSocialAccountUsername = ( state ) =>
-	get( state, 'login.socialAccount.username', null );
+	state.login.socialAccount?.username ?? null;
```

Comment:
`socialAccount` always contains a valid object. It just sometimes doesn't have the `username` property. That means the optional chaining is not needed, and the fallback is needed.

Reply:
Done.